games-roguelike/moria/moria-5.7.10.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils cmake-utils
+
+DESCRIPTION="Rogue-like D&D curses game similar to nethack (BUT BETTER)"
+HOMEPAGE="https://umoria.org/"
+SRC_URI="https://github.com/dungeons-of-moria/umoria/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND=">=sys-libs/ncurses-6:0"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	dev-util/cmake
```

Comment:
Doesn't cmake-utils take care of this?

Reply:
Thanks, I didn't know that. I've looked into cmake-utils.eclass, it does take care of BDEPEND.
Should we not set BDEPEND explicitly if it is set via eclass? I haven't found anything about that in devmanual.

Reply:
IMHO it's just repetition, and quite obvious that these build-system eclasses will provide the build-tool for DEPEND. But I mentioned it because @Zlogene always says to remove them, unless specific version is needed :)